Dockerfile

```diff
@@ -44,7 +44,7 @@ FROM vfabric as runner
 
 # Install Groovy
 WORKDIR /usr/bin
-RUN wget http://dl.bintray.com/groovy/maven/groovy-binary-2.3.0-beta-2.zip
+RUN wget https://repo1.maven.org/maven2/org/codehaus/groovy/groovy-binary/2.3.0/groovy-binary-2.3.0.zip
```

Comment:
the next line refers to the file name in this line, so `unzip` will fail because the file has been renamed from `groovy-binary-2.3.0-beta-2.zip` to `groovy-binary-2.3.0.zip`